examples/aws_cli.py

```diff
@@ -161,5 +162,6 @@ def ListImages(args: 'argparse.Namespace') -> None:
   images = aws_account.ListImages(qfilter)
 
   for image in images:
-    print('Name: {0:s}, ImageId: {1:s}'.format(image['Name'],
-                                               image['ImageId']))
+    print('Name: {0:s}, ImageId: {1:s}, '
+          'Location: {2:s}'.format(image['Name'], image['ImageId'],
+                                   image['ImageLocation']))
```

Comment:
Put the string on one line, and the parameters to format in an indented line:

```suggestion
    print('Name: {0:s}, ImageId: {1:s}, Location: {2:s}'.format(
        image['Name'], image['ImageId'], image['ImageLocation']))
```